skeleton/src/main/java/com/decathlon/vitamin/skeleton/VitaminSkeletonRectangularView.kt

```diff
@@ -0,0 +1,74 @@
+package com.decathlon.vitamin.skeleton
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Paint
+import android.util.AttributeSet
+import android.view.View
+import androidx.annotation.ColorInt
+import androidx.core.content.ContextCompat
+
+class VitaminSkeletonRectangularView @JvmOverloads constructor(
+    context: Context, attrs: AttributeSet? = null, defStyleAttr: Int = 0
+) : View(context, attrs, defStyleAttr) {
+
+    private val paint = Paint()
+
+    @ColorInt
+    var color: Int = 0
+        set(value) {
+            field = value
+            invalidate()
+        }
+
+    // Corner radius in pixel size
+    var cornerRadius: Int = 0
+        set(value) {
+            field = value
+            invalidate()
+        }
+
+    init {
+        val attributes = context.obtainStyledAttributes(
+            attrs,
+            R.styleable.VitaminSkeletonRectangularView,
```

Comment:
I think that if the style isn't in parameter of the constructor, we can't override it. (same comment for the rounded version)

Reply:
Conclusion: Allow style approach in addition to standalone approach